packages/webapp/pages/onboarding.tsx

```diff
@@ -159,6 +161,19 @@ export function OnboardPage(): ReactElement {
   };
 
   const onClickCreateFeed = () => {
+    if (activeScreen === OnboardingStep.EditTag) {
+      const onboardingContentType = getFeatureValue(
+        feature.onboardingContentType,
+      );
```

Comment:
Enrolment only happens when user is in edit tag page and has clicked the continue button